src/main/scala/is/hail/table/Table.scala

```diff
@@ -80,6 +80,8 @@ object Table {
     else if (!path.endsWith(".kt") && !path.endsWith(".kt/"))
       fatal(s"key table files must end in '.kt', but found '$path'")
 
+    GenomeReference.importReferences(hc.hadoopConf, path + "/references/")
```

Comment:
Is it possible that `path` already ends with a "/"? From the assertion at the top of the method, it seems like we're allowing that.

Reply:
the Hadoop file API (and most other file system APIs like UNIX) treat double forward slashes the same as single forward slashes, I think so that you can write stuff like this without having to check everywhere.

Reply:
oh cool, I didn't realize that

Reply:
me neither until Cotton pointed it out at some point! But it totally makes sense.